docroot/modules/features/resource/config/optional/views.view.mirador_view.yml

```diff
@@ -5,6 +5,7 @@ dependencies:
     - core.entity_view_mode.node.mirador
     - node.type.resource
     - user.role.administrator
+    - user.role.anonymous
```

Comment:
modified it in https://github.com/NVLI/nvli-main-website/pull/88/commits/6fe67993996517923a46670110dcc6df68842154